src/js/video.js

```diff
@@ -500,5 +509,36 @@ videojs.defineLazyProperty = defineLazyProperty;
 // In a major update this could become the default text and key.
 videojs.addLanguage('en', {'Non-Fullscreen': 'Exit Fullscreen'});
 
+const helpers = {
+  browser,
+  buffer: Buffer,
+  clamp,
+  computedStyle,
+  createLogger,
+  defineLazyProperty,
+  domData: DomData,
```

Comment:
this property is specifically not exported because it's contains internal proprties